app/forms/candidate_interface/contact_details_form.rb

```diff
@@ -6,7 +6,8 @@ class ContactDetailsForm
                   :address_line4, :postcode, :address_type, :country, :international_address
 
     validates :address_line1, :address_line3, :postcode, presence: true, on: :address, if: :uk?
-    validates :international_address, presence: true, on: :address, if: :international?
+    validates :address_line1, presence: true, on: :address, if: ->(form){ form.international? && FeatureFlag.active?(:international_addresses) }
```

Comment:
Do we intentionally only check the first line of international addresses? Presumably we're less strict because we can't be sure of the format?

Reply:
Yea we're really not sure what is and is not valid for each individual country so we decided this was probably the best approach.